database1/pgQueryTests.js

```diff
@@ -0,0 +1,64 @@
+const pg = require('./pgQueryTests.js')
+const { Client } = require('pg');
+
+const client = new Client({
+  user: 'clairemelbourne',
```

Comment:
Probably best practice to abstract this away into something like a .env file when you get the chance. It's just a username, but the principle is to not commit anything that could be dangerous, so imagine it was a password, API key, etc.

Reply:
woof good point. thank you